### PR TITLE
Added new id field to PacketStreamRequest and PacketStreamResponse messages in dataplane_interface.proto

### PR DESCRIPTION
**Pull request by dushyantarora**
This will allow clients to send multiple packets and be able to correlate output packets to input packets; thus improving testing and debuggability.

### CODE REVIEW

File: targets/simple_switch_grpc/switch_runner.cpp [hunk 1]
```diff
@@ -153,17 +175,10 @@ class DataplaneInterfaceServiceImpl
   }
 
   void transmit_fn_(port_t port_num, const char *buffer, int len) override {
```
Comment (antoninbas) [inline targets/simple_switch_grpc/switch_runner.cpp]: please call `my_transmit_fn` to make it a valid `DevMgrIface` implementation

File: targets/simple_switch_grpc/switch_runner.h [hunk 2]
```diff
@@ -86,6 +88,7 @@ class SimpleSwitchGrpcRunner {
   bm::DevMgrIface::port_t cpu_port;
   std::string dp_grpc_server_addr;
   int dp_grpc_server_port;
+  DataplaneInterfaceServiceImpl* dp_service;
```
Comment (antoninbas) [inline targets/simple_switch_grpc/switch_runner.h]: move `*`

File: targets/simple_switch/simple_switch.h [hunk 3]
```diff
@@ -111,10 +112,15 @@ class SimpleSwitch : public Switch {
   // returns the number of microseconds elasped since the clock's epoch
   uint64_t get_time_since_epoch_us() const;
 
+  static packet_id_t get_packet_id() {
```
Comment (antoninbas) [inline targets/simple_switch/simple_switch.h]: please add a comment for this function, and mention that it is not thread-safe

File: targets/simple_switch_grpc/switch_runner.cpp (hunk 1 above)
Comment (dushyantarora) [reply targets/simple_switch_grpc/switch_runner.cpp]: Done

File: targets/simple_switch/simple_switch.h [hunk 4]
```diff
@@ -111,10 +112,16 @@ class SimpleSwitch : public Switch {
   // returns the number of microseconds elasped since the clock's epoch
   uint64_t get_time_since_epoch_us() const;
 
+  // returns the packet id of most recently received packet. Not thread-safe.
+  static packet_id_t get_packet_id() {
```
Comment (antoninbas) [inline targets/simple_switch/simple_switch.h]: should this be (packet_id - 1) then to match the comment?

Comment (dushyantarora) [reply targets/simple_switch/simple_switch.h]: Fixed

**Comment by antoninbas** [comment]
@dushyantarora do you know why git shows me as the author of this change after merging it into master (https://github.com/p4lang/behavioral-model/commit/68eae8b340ad80d36b3b9e25fbba29f05da6ab8e)? I followed the same steps as usual... Anything special about how you pushed your changes for this PR?

**Comment by dushyantarora** [comment]
I followed the same steps I always do as well. I wont worry about it.

**Pull request merged**